app/src/main/java/com/nimontoy/android/view/adapter/viewholder/EventCardViewHolder.kt

```diff
@@ -4,16 +4,23 @@ import android.view.View
 import android.widget.TextView
 import com.nimontoy.android.R
 import com.nimontoy.android.model.Data
+import com.nimontoy.android.model.mypage.EventCard
+import com.nimontoy.android.view.custom.EventCardCellView
 import com.nimontoy.android.viewmodel.cell.DataCellViewModel
 
 class EventCardViewHolder (itemView: View) : DataViewHolder(itemView) {
-
-    val title_event_card: TextView = itemView.findViewById(R.id.title_event_card)
+    private val EventCardCell by lazy { itemView.findViewById<EventCardCellView>(R.id.event_card) }
```

Comment:
이제부터 findViewById로 아이템을 찾는 방법 대신 ktx 문법으로 직접 뷰에 접근해 아이디를 갖고 오는방법이 어떨까요?

Reply:
변수명도 왜 대문자 시작이죠?

Reply:
ktx 문법으로 했을 때 "Unresolved reference. None of the following candidates is applicable because of receiver type mismatch" 라는 오류가 뜹니다..
변수명 첫글자 소문자로 변경했습니다